app/products/calls/connection/connection.ts

```diff
@@ -201,8 +199,34 @@ export async function newConnection(
             }
         }
 
-        InCallManager.start({media: 'video'});
-        setSpeakerPhone(true);
+        InCallManager.start();
```

Comment:
Unrelated to the changes but should we plan to refactor this into a calls client class at some point? It feels like there's too much state to be still a function.

Reply:
Yah we can think about it -- I know more state -> lean towards classes, but not sure if there's much of a benefit in this case (there's nothing a class would give us, since we're not an event emitter).

Reply:
Yeah, it just feels like we are abusing the function because we are literally initiating a bunch of stuff and returning an object with all the methods.

Reply:
Yep, I'm not against it, I just figured it doesn't actually matter because javascript.

Reply:
Yes, that's the reason we got to this point and it will only get worse going forward as we add more to it. I created  https://mattermost.atlassian.net/browse/MM-52386 to track this, it's clearly nothing urgent, more like for rainy days.